pkg/virtual/apiexport/controllers/apireconciler/apiexport_apireconciler_controller.go

```diff
@@ -143,16 +148,17 @@ func (c *APIReconciler) enqueueAPIResourceSchema(obj interface{}) {
 	for _, obj := range exports {
 		export := obj.(*apisv1alpha1.APIExport)
 		klog.V(2).Infof("Queueing APIExport %s|%s for APIResourceSchema %s", clusterName, export.Name, name)
-		c.enqueueAPIExport(obj)
+		c.enqueueAPIExport(obj, logger, "reason", "APIResourceSchema change", "apiResourceSchema", name)
```

Comment:
pattern elsewhere is

```go
logger = logging.WithObject(logger, obj.(*apisv1alpha1.APIResourceSchema))
for _, export := range exports {
    c.enqueueAPIExport(obj, logger.WithValues("reason", "APIResourceSchema change"))
}
```

or some string-suffix thing when there are multiple stacked reasons

Reply:
Fix in next commit